packages/graph/hash_graph/tests/integration/postgres/entity.rs

```diff
@@ -70,6 +72,8 @@ async fn query() {
         .await
         .expect("could not create entity");
 
+    let identifier = metadata.identifier();
+
     let queried_organization = api
         .get_entity(identifier.entity_id())
```

Comment:
Rather than the intermediary identifier above can't we just call `metadata.identifier().entity_id()` here?

Reply:
Addressed in https://github.com/hashintel/hash/pull/1199/commits/6540ba6581ef0ef574a30e4d7b51ef7abf97a9a9

Reply:
I was saying we could do that all across the file to avoid the additional variables we added over the place :)

Reply:
Understood, have addressed in https://github.com/hashintel/hash/pull/1199/commits/7d985846626654fe3b6f0f5da545a963c394aeae